editor/property_editor.py

```
@@ -1766,6 +1766,67 @@ def on_parent_mover_changed(text):
+                    self.editor.update_all_ui()
+
+            portal_attach_cb.toggled.connect(on_portal_attach_toggled)
+
+            def on_portal_parent_mover_changed(text):
+                clean = text.replace(" (missing)", "")
+                if clean == "(none)":
+                    thing.properties['parent_mover'] = ''
+                    thing.properties['parent_offset'] = [0.0, 0.0, 0.0]
+                else:
+                    thing.properties['parent_mover'] = clean
+                    # Compute offset = portal pos − mover pos
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)